### PR TITLE
burrow cash pool stats

### PR DESCRIPTION
**Pull request by zavodil**

### CODE REVIEW

File: src/adaptors/burrow/index.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    const target_list = [];
+    burrowPoolStats.map(item => {
+        const target = {
+            pool: 'burrow-pool-' + item.token_id,
+            chain: 'NEAR',
+            project: 'burrow',
+            symbol: item.symbol,
+            tvlUsd: item.tvlUsd,
+            apyReward: item.apyReward,
```
Comment (ciocan) [inline src/adaptors/burrow/index.ts]: add apyRewardTvl to rewards

```
            apyReward: item.apyReward + item.apyRewardTvl,
```

**Comment by slasher125** [comment]
sers, any chance we can get this info from on chain/subgraph instead of api?

**Comment by zavodil** [comment]
@slasher125 The api is loading all data from chain https://github.com/NearDeFi/burrow-api/commit/cc51bc4fcbec13cd77cb0c4427dce2fabbb7403a 
It includes some internal functions from the main app.
Is it acceptable for you?

**Comment by llamatester** [comment]
The burrow adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       95 passed, 95 total
    Snapshots:   0 total
    Time:        0.58 s
    Ran all test suites.
    
    Nb of pools: 13
     
    
    Sample pools:
    ┌─────────┬────────────────────────────────────────────────────────────────────────────┬────────┬──────────┬──────────┬────────────────────┬────────────────────┬────────────────────────┬────────────────────────────────────────────────────────────────────┬──────────────────────────────────────┐
    │ (index) │                                    pool                                    │ chain  │ project  │  symbol  │       tvlUsd       │     apyReward      │        apyBase         │                          underlyingTokens                          │             rewardTokens             │
    ├─────────┼────────────────────────────────────────────────────────────────────────────┼────────┼──────────┼──────────┼────────────────────┼────────────────────┼────────────────────────┼────────────────────────────────────────────────────────────────────┼──────────────────────────────────────┤
    │    0    │ 'burrow-pool-a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48.factory.bridge.near' │ 'NEAR' │ 'burrow' │  'USDC'  │ 7093102.284523582  │ 3.5202616957616484 │  0.16766594933594528   │ [ 'a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48.factory.bridge.near' ] │       [ 'token.burrow.near' ]        │
    │    1    │                        'burrow-pool-meta-pool.near'                        │ 'NEAR' │ 'burrow' │ 'STNEAR' │ 5744384.193448567  │ 3.5202616957616484 │  0.12430946866629632   │                        [ 'meta-pool.near' ]                        │       [ 'token.burrow.near' ]        │
    │    2    │ 'burrow-pool-dac17f958d2ee523a2206206994597c13d831ec7.factory.bridge.near' │ 'NEAR' │ 'burrow' │ 'USDT.e' │ 5179769.503405741  │ 3.5202616957616484 │  0.032566555947025365  │ [ 'dac17f958d2ee523a2206206994597c13d831ec7.factory.bridge.near' ] │       [ 'token.burrow.near' ]        │
    │    3    │                     'burrow-pool-linear-protocol.near'                     │ 'NEAR' │ 'burrow' │ 'LINEAR' │ 3877547.4835716463 │ 3.5202616957616484 │ 0.00011229316023041015 │                     [ 'linear-protocol.near' ]                     │       [ 'token.burrow.near' ]        │
    │    4    │                          'burrow-pool-wrap.near'                           │ 'NEAR' │ 'burrow' │ 'wNEAR'  │ 2797266.151006743  │ 3.5202616957616484 │   3.4567638698981917   │                          [ 'wrap.near' ]                           │       [ 'token.burrow.near' ]        │
    │    5    │ 'burrow-pool-2260fac5e5542a773aa44fbcfedf7c193bc2c599.factory.bridge.near' │ 'NEAR' │ 'burrow' │  'WBTC'  │ 2273072.8945216406 │ 3.5202616957616484 │  0.29809767835551476   │ [ '2260fac5e5542a773aa44fbcfedf7c193bc2c599.factory.bridge.near' ] │       [ 'token.burrow.near' ]        │
    │    6    │                            'burrow-pool-aurora'                            │ 'NEAR' │ 'burrow' │  'ETH'   │ 941862.1429035666  │ 3.5202616957616484 │   0.3668467288289265   │                            [ 'aurora' ]                            │       [ 'token.burrow.near' ]        │
    │    7    │ 'burrow-pool-6b175474e89094c44da98b954eedeac495271d0f.factory.bridge.near' │ 'NEAR' │ 'burrow' │  'DAI'   │ 421958.8842992078  │ 3.5202616957616484 │   0.4646309689901019   │ [ '6b175474e89094c44da98b954eedeac495271d0f.factory.bridge.near' ] │       [ 'token.burrow.near' ]        │
    │    8    │ 'burrow-pool-4691937a7508860f876c9c0a2a617e7d9e945d4b.factory.bridge.near' │ 'NEAR' │ 'burrow' │  'WOO'   │ 294563.5497884211  │ 3.5202616957616484 │ 0.0006782395197473845  │ [ '4691937a7508860f876c9c0a2a617e7d9e945d4b.factory.bridge.near' ] │       [ 'token.burrow.near' ]        │
    │    9    │                      'burrow-pool-token.burrow.near'                       │ 'NEAR' │ 'burrow' │  'BRRR'  │ 179091.33899013986 │ 73.32031886484808  │           0            │                      [ 'token.burrow.near' ]                       │ [ 'wrap.near', 'token.burrow.near' ] │
    └─────────┴────────────────────────────────────────────────────────────────────────────┴────────┴──────────┴──────────┴────────────────────┴────────────────────┴────────────────────────┴────────────────────────────────────────────────────────────────────┴──────────────────────────────────────┘

**Comment by slasher125** [comment]
> @slasher125 The api is loading all data from chain [NearDeFi/burrow-api@cc51bc4](https://github.com/NearDeFi/burrow-api/commit/cc51bc4fcbec13cd77cb0c4427dce2fabbb7403a) It includes some internal functions from the main app. Is it acceptable for you?

k, lets leave it then.
I've just realised u are a lending protocol. we got lots of new pages (optimizer, strategies, etc.) utilizing lending/borrow data. so, to be integrated on those pages make sure to add some additional fields:
```
  // optional lending protocol specific fields:
  apyBaseBorrow?: number;
  apyRewardBorrow?: number;
  totalSupplyUsd?: number;
  totalBorrowUsd?: number;
  ltv?: number;
```

**Comment by slasher125** [comment]
gonna merge this one for now. feel free to open a new PR with the above extra fields

**Pull request merged**